src/sequelize.js

```diff
@@ -1037,10 +1038,23 @@ Use Sequelize#query if you wish to use replacements.`);
   }
 
   // TODO: rename to getDatabaseVersion
```

Comment:
@ephys I think we can get rid of this now?

Reply:
yep